wetterdienst/core/scalar/export.py

```diff
@@ -166,12 +166,12 @@ def to_target(self, target: str):
             elif target.endswith(".parquet"):
                 """
                 # Acquire data and store to Parquet file.
-                alias fetch="wetterdienst dwd observations values --station=1048,4411 --parameter=kl --resolution=daily --period=recent"
-                fetch --target="file://observations.parquet"
+                alias fetch="wetterdienst dwd observation values --station=1048,4411 --parameter=kl --resolution=daily --period=recent"
```

Comment:
I see. This inline documentation has be overhauled, as it still shows the old syntax.